hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtil.java

```diff
@@ -977,6 +1000,28 @@ public SingleProcessHBaseCluster getMiniHBaseCluster() {
       hbaseCluster + " not an instance of " + SingleProcessHBaseCluster.class.getName());
   }
 
+  /**
+   * Stops mini hbase, zk, and hdfs clusters.
+   * If zk or hdfs is external, clean the znode or dfs path.
+   * @see #startMiniCluster(int)
+   */
+  public void shutdownMiniCluster(StartTestingClusterOption option) throws IOException {
```

Comment:
I think we should know whether we have started DFS cluster and zk by our own? Or at least we can store the StartTestingClusterOption so users do not need to pass it again?